README changes (#49)
NexTre-dev/retro-gadgets-docs: en/README.md

Retro Gadgets is a gadget creation station where you invent, build, solder, code and customise your own electrifying gadgets then share them with the world.

This repository contains the documentation for doing anything code-related in Retro Gadgets.  
You can find [the official documentation here](https://docs.retrogadgets.game), although it might not be of much use (yet)

* [Buy the game here!](https://store.steampowered.com/app/1730260/Retro_Gadgets/)
* [Join the Official Discord Server!](https://discord.com/invite/np6FUDjt9E)
* [Learn how you can contribute by opening a pull request!](./docs/github/Contributing.md)

### Coding gadgets

> Retro Gadgets uses Luau to code gadgets, which is basically Lua but it [has some differences](https://luau-lang.org/compatibility). You can still use resources for learning Lua and apply that knowledge to Luau. Here are some places to learn Lua (and Luau):

Reviewer: Slight grammatical change, feel free to resolve this chain if you'd like to ignore it.
```suggestion
Retro Gadgets uses Luau to code gadgets, which is basically Lua but [with some differences](https://luau-lang.org/compatibility). You can still use resources for learning Lua and apply that knowledge to Luau. Here are some places to learn Lua (and Luau):
```

* [Lua documentation](https://www.lua.org/start.html)
* [Learn X in Y minutes, where X=Lua](https://learnxinyminutes.com/docs/lua/)
* [Roblox Luau documentation](https://create.roblox.com/docs/scripting/luau)
* [Luau documentation](https://luau-lang.org/library)

## Okay... how do I get started?
Well, [buy the game](https://store.steampowered.com/app/1730260/Retro_Gadgets/) if you haven't done that already.
* [Getting Started](./examples/getting-started.md)

## Todo
- [X] Finish README.md
  - [X] Add proper description about programming in Retro Gadgets
  - [X] Add resources for learning Lua

> - [ ] API Documentation

Reviewer: Actually, just noticed, I think this should be checked according to #30 
![image](https://user-images.githubusercontent.com/31492625/217383434-7a608d45-5796-4f1d-8ac4-88147cb43452.png)
```suggestion
- [X] API Documentation
```
